client/js/widgets/widget.js

```diff
@@ -1496,11 +1496,17 @@ export class Widget extends StateManaged {
           const turn = indexList[mod(nextTurnIndex, indexList.length)];
 
           collections[a.collection] = [];
-          //saves turn into all seats and creates output collection with turn seats
-          for(const w of c) {
-            await w.set('turn', w.get('index') == turn);
-            if(w.get('turn') && w.get('player'))
-              collections[a.collection].push(w);
+
+          if(readonly) {
```

Comment:
probably this should be readOnly

also didn't test but why doesn't this need to be a.readOnly like every property in a routine?

Reply:
done